packages/gcf-utils/test/gcf-utils.ts

```diff
@@ -212,4 +214,156 @@ describe('GCFBootstrapper', () => {
       sinon.assert.calledOnce(configStub);
     });
   });
+
+  describe('getProbotConfig', () => {
+    let bootstrapper: GCFBootstrapper;
+    let secretClientStub: v1.SecretManagerServiceClient;
+    let secretsStub: sinon.SinonStub;
+    let secretVersionNameStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      secretClientStub = new v1.SecretManagerServiceClient();
+      bootstrapper = new GCFBootstrapper(secretClientStub);
+
+      secretVersionNameStub = sinon
+        .stub(bootstrapper, 'getLatestSecretVersionName')
+        .callsFake(() => {
+          return 'foobar';
+        });
+    });
+
+    afterEach(() => {
+      secretsStub.reset();
+      secretVersionNameStub.reset();
+    });
+
+    it('gets the config', async () => {
+      secretsStub = sinon
+        .stub(secretClientStub, 'accessSecretVersion')
+        .callsFake(() => {
+          return Promise.resolve([
```

Comment:
Instead of using `.callsFake` like this, you can just say `.resolves([{ payload ...`

Reply:
Done.